invenio_vocabularies/contrib/awards/schema.py

```diff
@@ -61,6 +61,12 @@ class AwardSchema(BaseVocabularySchema, ModePIDFieldVocabularyMixin):
 
     organizations = fields.List(fields.Nested(AwardOrganizationRelationSchema))
 
+    description = fields.String()
+
+    start_date = fields.String()
+
```

Comment:
Do we need to add these in `AwardRelationSchema` as well?

Reply:
I would say no for now, since we don't have a use-case where we want to query records by their awards' start/end dates...